### DISCUSSION TITLE
Program accessing files in sandbox automatically gets sandboxed now?

### DISCUSSION THREAD

**Discussion by wshyang** [sandboxie-plus/Sandboxie#1807] [Q&A Feedback]
I was previously running a very old build of Sandboxie and only just updated to the current version today, so this seems to be a new feature.

I've been noticing that now when I try to open a file through Windows explorer or Command Prompt from the storage path of a Sandbox, the program will automatically get intercepted and be run inside the Sandbox as well.

Is it possible to disable this new behaviour?

Just to explain why I want this behaviour to be disabled... for a Sandbox in particular, I tend to back up the contents of the Sandbox from a ramdisk to a non-volatile disk from time to time using a script, and the script calls Robocopy. So I noticed that when the script changes to the Sandbox directory and calls Robocopy, Robocopy now ends up running within the Sandbox.

This behavious hence is causing me a bit of problems, heh

**Comment by DavidXanatos** [comment]
Emm... yea i guess you were using the free version, what is interfering with your usage scheme is the forced folders feature, which  previously was only available in the payed version now its free, you can use the disable force programs option from the box or tray menu.

The force folders take a list of folders but also by default include the box paths, and its not intended to remove those you can only disable this globally for a set periode of time.

**Reply by isaak654** [reply]
See also #1618 if you belong to the same use case.
